testing/py-graphviz/APKBUILD

```diff
@@ -1,17 +1,17 @@
 # Contributor: Leonardo Arena <rnalrd@alpinelinux.org>
 # Maintainer:
 pkgname=py-graphviz
-pkgver=0.8.1
+_pkgname=${pkgname#py-}
+pkgver=0.8.3
 pkgrel=0
 pkgdesc="Simple interface for Graphviz"
 url="https://github.com/xflr6/graphviz"
 arch="noarch"
 license="MIT"
-depends=""
+options="!check" # Depends on pytest-cov
```

Comment:
#4655 has been merged. Can check be enabled now?

Reply:
Now it needs https://pypi.org/project/pytest-mock/

Reply:
with a package from https://github.com/alpinelinux/aports/pull/5310 version 0.9 passes tests locally